samples/swift/README.md

```diff
@@ -0,0 +1,9 @@
+Swift Samples
+-----
+
+This directory contains a collection of swift code samples. 
+
+In order to run the project you'll need a valid app in Firebase and 
```

Comment:
Include a little bit more setup instructions (e.g. go to console.firebase.google.com, setup a app with bundle id <whatever it is>, enable any auth providers required etc. )